base/commands/viridian/custom_class_upload.go

```diff
@@ -0,0 +1,54 @@
+package viridian
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hazelcast/hazelcast-commandline-client/clc"
+	. "github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
+)
+
+type CustomClassUploadCmd struct{}
+
+func (cmd CustomClassUploadCmd) Init(cc plug.InitContext) error {
+	cc.SetCommandUsage("upload-custom-class [cluster-ID/name] [path-to-class] [flags]")
```

Comment:
```suggestion
	cc.SetCommandUsage("upload-custom-class [cluster-ID/cluster-name] [path-to-class] [flags]")
```
For consistency with other commands.